hw/dv/tools/ralgen/README.md

```diff
@@ -19,6 +19,15 @@ generator, the following parameters are set:
   which the register classes are derived. Set this option to derive the register
   classes not from the default `dv_base_reg`, but from user defined custom
   class definitions.
+  This argument follows the following format:
+  `--dv-base-prefix block:type:dv_base_name,block:type:dv_base_name`.
+  `block`: can be any block names, or use `default` to override all blocks
+  within the hjson.
+  `type`: can be `reg_block`, `reg`, `reg_field`, `reg_pkg`, or use `all` to
```

Comment:
I assume this operation will change the register base address for some specific items in RAL. But what does it mean for a field to change base address? Should this only apply down to a register, since that is the smallest addressable item in RAL?

Reply:
Thanks for the inputs! In ral gen, the field base class is dv_base_reg_field. I think the usecase to update a field base class would be: for example, ast override some dv_base_reg_field class to ast_base_reg_field class, then we only need to update it by: ast:reg_field:ast_base_reg_field.

Reply:
This does not modify the addresses. It only changes the base class from which the auto-generated RAL classes extend from.

Reply:
I see, thanks for the clarification.